Nodejs/Tests/TestAdapter.Tests/TestAdapter.Tests.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Why not .NET 5? 😄

Reply:
Oh! no reason I just didn't have it installed. Upgraded.